repository_files.go

```diff
@@ -99,7 +99,7 @@ func (s *RepositoryFilesService) GetRawFile(pid interface{}, fileName string, op
 	if err != nil {
 		return nil, nil, err
 	}
-	u := fmt.Sprintf("projects/%s/repository/files/%s/raw", url.QueryEscape(project), url.QueryEscape(fileName))
+	u := fmt.Sprintf("projects/%s/repository/files/%s/raw", url.QuesryEscape(project), url.PathEscape(fileName))
```

Comment:
There a typo in here: `QuesryEscape` (twice actually)

Reply:
Oh sorry, i've just fixed that.